content/en/docs/Customizing/upgrades.md

```diff
@@ -38,7 +38,6 @@ Specifically, it allows to configure:
 - **VERIFY**: Turns on and off image verification. Currently available for official `cOS` release channels
 - **RECOVERY_IMAGE**: Allows to specify a different image for the recovery partition. Similarly to **UPGRADE_IMAGE** needs to be either an image reference or a package.
 
-An example on how to tweak this file via cloud-init is available [here](https://github.com/rancher-sandbox/cos-toolkit-sample-repo/blob/7355876847367b75485873987e1217f1e1fe6254/packages/sampleOS/02_upgrades.yaml#L41)
```

Comment:
We can link directly to the cOS ones here https://github.com/rancher-sandbox/cOS-toolkit/blob/master/packages/cloud-config/oem/02_upgrades.yaml

Reply:
ah. was already linked above, nevermind!